subsys/bluetooth/controller/CMakeLists.txt

```
@@ -78,6 +78,9 @@ if(CONFIG_BT_LL_SW_SPLIT)
     CONFIG_BT_HCI_MESH_EXT
     ll_sw/ll_mesh.c
     )
+  zephyr_library_sources(
+    ll_sw/ull_iso_broadcast.c
+    )
```

Move into with broadcaster

```suggestion
  zephyr_library_sources_ifdef(
  CONFIG_BT_CTLR_ADV_ISO
    ll_sw/ull_adv_iso.c
    )
```

Move into with observer

```suggestion
  zephyr_library_sources_ifdef(
  CONFIG_BT_CTLR_SYNC_ISO
    ll_sw/ull_sync_iso.c
    )
```